pkg/identity/metricsd.go

```diff
@@ -72,7 +73,14 @@ func Metricsd(idConfig *config.IdentityConfig, stopChan <-chan struct{}) (error,
 
 	if idConfig.TargetDomainRoles != "" && idConfig.RoleCertDir != "" {
 		for _, domainrole := range strings.Split(idConfig.TargetDomainRoles, ",") {
-			exporter.Files = append(exporter.Files, strings.TrimSuffix(idConfig.RoleCertDir, "/")+"/"+domainrole+".cert.pem")
+			targetDomain, targetRole, err := athenz.SplitRoleName(domainrole)
+			if err != nil {
+				continue
+			}
+			// if RoleCertFilenameDelimiter = "_" then,
+			// fileName = your-domain_your-role.cert.pem"
```

Comment:
```suggestion
```
can remove the comment?

Reply:
You are right, a bit superfluous. but then I think it could be helpful for those who are not familiar with Athenz.

Reply:
nvm. will delete it.